examples/README.md

```diff
@@ -56,6 +56,7 @@ This page lists the available common switches with description. Unless stated ot
 | SLAAC | OT_SLAAC | Enables support for adding auto-configured SLAAC addresses by OpenThread. This feature is enabled by default. |
 | SNTP_CLIENT | OT_SNTP_CLIENT | Enables support for SNTP Client. |
 | SPINEL_ENCRYPTER_LIBS | not implemented | Specifies library files (absolute paths) for implementing the NCP Spinel Encrypter. |
+| SSED_SSED_LINK | OT_SSED_SSED_LINK | Enables the optional Thread 1.2 feature of SSED-to-SSED Link. |
```

Comment:
I'm thinking if we may add the doc after we submit the whole feature. Users may get confused: it's in the doc but cannot be used.

Reply:
Agree. We can add it to README after the feature is completed.